tensformation/main.go

```diff
@@ -77,12 +78,16 @@ func main() {
 		log.Fatalf("failed to create client, %v", err)
 	}
 
+	httpClient := http.Client{
+		Timeout: 500 * time.Second,
+	}
+
```

Comment:
We're implicitly setting this on tf-inference server side with the CLI arg for API timeout: https://github.com/mylesagray/anpr-knative/blob/master/manifest.yaml#L151

I have it disabled right now (`=0`), but we can set it to whatever - to me its less configurable if its in tensformation unless we expose it as a configurable via ENV or arg?

Is there value to having distinct timeouts on each side?

Reply:
I dont know if there is value or not.. By default, the go HTTP client sets a timeout of 0 ( or infinite) so if something goes wrong it will never give up trying if we do not set a value. I am okay with moving this to an ENV variable! Ill get it done today!

Reply:
I'm cool with this as long as it's an ENV var, and if we default ENV var to 0 - to me, if the inference server can control the timeout itself, it's better placed to decide if something has died

Reply:
Ive removed the timeout completely. i dont think we need it.